presto-jmx/src/main/java/com/facebook/presto/connector/jmx/JmxConnectorConfig.java

```diff
@@ -41,7 +42,28 @@
     @Config("jmx.dump-tables")
     public JmxConnectorConfig setDumpTables(String tableNames)
     {
-        this.dumpTables = ImmutableSet.copyOf(Splitter.on(',').omitEmptyStrings().trimResults().split(tableNames));
+        ImmutableSet.Builder<String> dupTables = ImmutableSet.builder();
+        String tableName = "";
+        Iterable<String> parts = Splitter.on(',').omitEmptyStrings().split(tableNames);
```

Comment:
How about:

```java
        this.dumpTables = Splitter.on(Pattern.compile("(?<!\\\\),")) // match "," not preceded by "\"
                .omitEmptyStrings()
                .splitToList(tableNames)
                .stream()
                .map(part -> part.replace("\\,", ",")) // unescape all escaped commas
                .collect(Collectors.toSet());
```

Basically, it splits on "," not preceded by "\", but it doesn't consider the preceding character as part of the split pattern (it uses regex negative look-behind).